src/main/java/com/autotune/database/table/KruizeExperimentEntry.java

```diff
@@ -56,12 +57,11 @@ public class KruizeExperimentEntry {
     private String performance_profile;
     @Enumerated(EnumType.STRING)
     private AnalyzerConstants.ExperimentStatus status;
-    @JdbcTypeCode(SqlTypes.JSON)
-    private JsonNode datasource;
+    private String datasource;
     @JdbcTypeCode(SqlTypes.JSON)
     private JsonNode extended_data;
-    @JdbcTypeCode(SqlTypes.JSON)
-    private JsonNode meta_data;
+    @OneToOne //TODO: to be updated in future
+    private KruizeMetadata metadata;
```

Comment:
Please refrain from utilizing this column. It has been designated for storing information such as "when, where, and who" details, but this functionality has not been implemented yet.

Instead, establish a separate FKey columns.

Reply:
Updated

Reply:
Apologies, I just realized that it can't be one-to-one. Since each experiment can involve one or more containers, I believe it should be one-to-many.